clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -314,6 +317,7 @@ export default async function handler(
       : ""
   }
   window.Fides.config = ${fidesConfigJSON};
+  window.Fides.version = "${version}";
```

Comment:
I'd prefer we do this as a build-time step in the `fides-js` project instead of here. Right now this means you'll only reliably get that version number when you pull from the privacy center, but in some prod & test sites we include the FidesJS script directly from it's `dist/` version.

You can do the exact same steps in the rollup build 👍